frontend/public/style/_layout.scss

```diff
@@ -6,6 +6,10 @@ body,
 .pf-c-page__main-section,
 .pf-c-drawer {
   height: 100%; // so .co-p-has-sidebar, .yaml-editor are full height
+
+  &__content {
+      z-index: 1;
```

Comment:
`z-index:2` and `z-index:1` seems a bit tight to me, do we have a pattern for this property or have we not yet needed one?

Reply:
Yes, maybe use units of 10
```
.co-notification-drawer {
  z-index: 10;
}
```